js/index.js

```diff
@@ -1 +1,71 @@
-// JS goes here
\ No newline at end of file
+class Hamburger{
+    constructor(element) {
+        this.element = element;
+        this.button = this.element.querySelector('.menu-button');
+        this.content = this.element.querySelector('.menu-content');
+        this.button.addEventListener('click', () => {
+            this.toggleContent();})
+    }//end constructor
+
+    toggleContent(){
+        
+//         let buttons = this.element.querySelectorAll('.menu-button');
+//         buttons.forEach(button => {
+//             button.classList.remove('selected');
+//         })
+//         // this.body.classList.toggle('overflow');
+        
+        this.content.classList.toggle('menu-hidden');
+//         // this.element.classList.toggle('selected');
+//         this.element.classList.toggle('selected');
+        
+    } //end toggleContent
+
+} // end Hamburger class
+
+let navMenu = document.querySelectorAll('.menu');
+navMenu = Array.from(navMenu).map(menu => new Hamburger(menu));
+
+// let body = document.querySelector('.overflow');
+// class Button{
+//     constructor(element){
+//         this.element = element;
+//         this.data = element.dataset.button;
+//         this.itemElement = document.querySelector(`.menu-button[data-button= "${this.data}"]`);
+       
+//     }//methods go here:
+
+// }//end Button class
+
+
+
+// Array.from(button).forEach(function(element){
+//     element.classList.remove('selected');
+// });
+//     this.element.classList.add('selected');
+// navMenu[1].toggleContent();
+
+
+const hamburgerButton = document.querySelector('.span');
+const navigation = document.querySelector('.menu-button');
+const overflows = document.querySelector('.overflow');
+let count = 0;
+hamburgerButton.addEventListener('click', () => {
+    if (count%2 === 0) {
```

Comment:
if you can figure out how to do this inside the component, that would be cool! You'd want to have a property on the instance that keeps track of whether it's open (`this.open` or something), and in the method, update the instance property, then toggle the class on the two images based on that. Also, have two images and have whether they show determined by a CSS class with `display: none`